system/extensions/esp32/wifi.toit

```diff
@@ -59,6 +66,12 @@ class WifiServiceDefinition extends NetworkServiceDefinitionBase:
     if module.ssid != ssid or module.password != password:
       throw "wifi already connected with different credentials"
 
+    if save:
+      if config:
+        store_.set WIFI_CONFIG_STORE_KEY config
+      else:
+        store_.delete WIFI_CONFIG_STORE_KEY
```

Comment:
This is a bit weird.
You would basically say `connect --save`, and we would connect to the saved configuration, but then remove it.